src/doc/rustc/src/codegen-options/index.md

```diff
@@ -31,8 +31,9 @@ Supported values can also be discovered by running `rustc --print code-models`.
 
 ## codegen-units
 
-This flag controls how many code generation units the crate is split into. It
-takes an integer greater than 0.
+This flag controls the maximum number of code generation units the crate is
+split into. (The exact number will depend on the size and structure of the
+crate's source code.) It takes an integer greater than 0.
```

Comment:
I could see the parenthetical statement being interpreted as "we may create more CGUs than the value you specified". Maybe this formulation is more clear on that point? 
 
```suggestion
split into. (Fewer code generation units may be created depending on the 
size and structure of the crate's source code.) It takes an integer 
greater than 0.
```

Reply:
I'll remove the parenthetical. The important part of the change is specifying that it's a maximum.